src/components/ReactionView/hooks/useReaction.ts

```diff
@@ -38,7 +43,30 @@ const useReaction = (props: ReactionViewProps) => {
   const isSinglePress = showPopupType === GlobalConstants.onPress;
 
   const isLongPress = showPopupType === GlobalConstants.default;
+  const panResponder = useRef(
+    PanResponder.create({
+      onMoveShouldSetPanResponder: () => true,
+      onStartShouldSetPanResponder: () => true,
+      onPanResponderGrant: () => {},
+      onPanResponderMove: event => onGesture(event),
+      onPanResponderRelease: () => {
+        setPosition(0);
```

Comment:
The same function is created in the useEmojiView.ts file. So can we optimize and imported from one file?

Reply:
we are not able to get the exact gesture component from which we started the gesture because here we are managing gestures on their respective component condition. so we can optimize it but then there we'll be other things to handle , so it is better not make it common